noaa/challenges/sql.json

```diff
@@ -0,0 +1,275 @@
+            "name": "increase-max-open-scoll-context",
+            "operation-type": "put-settings",
+            "body": {
+              "transient" : {
+                  "search.max_open_scroll_context" : "2000"
+              }
+            }
+          },
+          "tags": ["setup"]
+        },
+        {
+          "operation": {
+            "name": "select_const",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT 1"
+            }
+          },
+          {{ task_options_10qps }}
+        },
+        {
+          "operation": {
+            "name": "select_star",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT * FROM \"weather-data-2016\""
+            }
+          },
+          {{ task_options_10qps }}
+        },
+        {
+          "operation": {
+            "name": "select_field",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT station.name FROM \"weather-data-2016\""
+            }
+          },
+          {{ task_options_10qps }}
+        },
+        {
+          "operation": {
+            "name": "select_scriptField",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT LCASE(SUBSTRING(station.name, 0, 2)) FROM \"weather-data-2016\""
+            }
+          },
+          {{ task_options_10qps }}
+        },
+        {
+          "operation": {
+            "name": "select_stringScriptFilter",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT station.name FROM \"weather-data-2016\" WHERE LCASE(SUBSTRING(station.name, 0, 2)) = 'sa' AND station.elevation > 2800"
+            }
+          },
+          {{ task_options_1qps }}
+        },
+        {
+          "operation": {
+            "name": "select_stringScriptFilterWithLimit",
+            {# https://github.com/elastic/elasticsearch/issues/80523 #}
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT station.name FROM \"weather-data-2016\" WHERE LCASE(SUBSTRING(station.name, 0, 2)) = 'sa' AND station.elevation > 2800 LIMIT 10"
+            }
+          },
+          {{ task_options_1qps }}
+        },
+        {
+          "operation": {
+            "name": "select_numericScriptFilter",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT station.name FROM \"weather-data-2016\" WHERE CEIL(TMIN) % 7 = 5 AND station.elevation > 2800"
+            }
+          },
+          {{ task_options_5qps }}
+        },
+        {
+          "operation": {
+            "name": "select_stringScriptSort",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT station.name FROM \"weather-data-2016\" WHERE station.elevation > 2800 ORDER BY LCASE(SUBSTRING(station.name, 0, 2))"
+            }
+          },
+          {{ task_options_1qps }}
+        },
```

Comment:
Are there significant difference between numeric and string sorting?
And the same comment would apply to the rest of the numeric vs string operations.